components/pango_core/include/pangolin/utils/httplib.h

```diff
@@ -5,8 +5,7 @@
 //  MIT License
 //
 
-#ifndef CPPHTTPLIB_HTTPLIB_H
```

Comment:
I have a couple of third party headers like this I should move into a third-party folder so that they don't get formatted / changed.